addons/web_editor/static/lib/odoo-editor/test/spec/format.test.js

```diff
@@ -14,49 +14,72 @@ const strikeThrough = async editor => {
     await editor.execCommand('strikeThrough');
 };
 
-describe('Format', () => {
+describe.only('Format', () => {
+    const b = content => `<span style="font-weight: bolder;">${content}</span>`;
+    const notB = content => `<span style="font-weight: normal;">${content}</span>`;
```

Comment:
I don't really like these variables. I think it makes the html in the tests harder to read because you have to mentally insert their value in the html when reading it. However, I'm sure you think exactly the opposite otherwise you wouldn't have created them in the first place. ^^'

Reply:
Let me just give you an example of what one of the tests looked like without it:

```js
`<p>ab<span style="text-decoration-line: underline;"><span style="font-style: italic;">cd</span></span><span style="font-style: italic;"><span style="text-decoration-line: underline;">ghi</span><span oe-zws-empty-inline="">​[]</span></span><span style="text-decoration-line: underline;"><span style="font-style: italic;">ef</span></span></p>`
```

I just figured this was easier to edit, read and write:

```js
<p>ab${u(i(`cd`))}${i(u(`ghi`) + `<span oe-zws-empty-inline="">\u200b[]</span>`)}${u(i(`ef`))}</p>
```

Reply:
Really? Would you prefer
```js
'<p>ab' +
    u(
        i('cd')
    ) +
    i(
        u('ghi') +
        '<span oe-zws-empty-inline="">\u200b[]</span>'
    ) +
    u(
        i('ef')
    )
</p>
```
?